src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -23,8 +23,16 @@
 from sentry.utils import metrics
 
 from .constants import CRASH_SESSIONS_DISPLAY
-from .health_checks import is_crash_free_rate_healthy_check, is_error_count_healthy
-from .utils import fetch_sessions_data, get_errors_counts_timeseries_by_project_and_release
+from .health_checks import (
+    is_crash_free_rate_healthy_check,
+    is_error_count_healthy,
+    is_new_issue_count_healthy,
+)
+from .utils import (
+    fetch_sessions_data,
+    get_errors_counts_timeseries_by_project_and_release,
+    get_new_issue_counts,
+)
```

Comment:
This always ends up happening when the IDE does the import, it uses the last used convention haha
Should we do relative paths or keep to regular full imports?

Reply:
🤔 last used?
i think this is just alphabetical right?

Reply:
Ah, i meant the `.health_checks` and the `.utils` instead of `src.blah.foo.health_checks`. "Last used" here meant the previous import in this file was using relative imports already, check out `.constants`, so I think the IDE was like "they want relative imports" haha

Reply:
Ah i see 😅 
actually i just decided to use relative imports just because it was easier on the eyes lol

I don't really have any strong opinions and agree absolutes are probs better and typically suggested lol 😬 
I can update that heh